### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by davideschiera**
More notes (not considered in these changes):

- `Recent updates`: seems like a changelog, I'd put it at the end or in a separate file
- `Stop and Start`: hard to read (initial description + various scripts)
- You shouldn't need `a` elements for "anchors"
- Sysdig Cloud, sdc or similar references should be replaced with "Sysdig"
- tools/projects names should be properly capitalized or `coded`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,35 +186,37 @@ Describe the sdc-collector service to see the full collector endpoint URL. It wi
...
     `kubectl create -f etc/sdc-config.yaml`
 
-3. Create Kubernetes StorageClasses identifying the types of disks to be provided to our datastores.
+3. Create Kubernetes StorageClasses identifying the types of disks to be provided to our datastores
 
     `kubectl create -R -f datastores/storageclasses/`
 
 4. Creates the datastore statefulsets (redis, mysql, elasticsearch and cassandra). Elasticsearch and Cassandra are 
-automatically setup with --replica=3 generating full clusters. Redis and mysql are configured with master/slave replication. 
+automatically setup with --replica=3 generating full clusters. Redis and MySQL are configured with master/slave replication.
```
Comment (mbreitung) [inline README.md]: this is wrong. we don't have HA for mysql and redis yet

File: README.md [hunk 2]
```diff
@@ -179,35 +186,37 @@ Describe the sdc-collector service to see the full collector endpoint URL. It wi
...
-3. Create Kubernetes StorageClasses identifying the types of disks to be provided to our datastores.
+3. Create Kubernetes StorageClasses identifying the types of disks to be provided to our datastores
 
     `kubectl create -R -f datastores/storageclasses/`
 
 4. Creates the datastore statefulsets (redis, mysql, elasticsearch and cassandra). Elasticsearch and Cassandra are 
-automatically setup with --replica=3 generating full clusters. Redis and mysql are configured with master/slave replication. 
+automatically setup with --replica=3 generating full clusters. Redis and MySQL are configured with master/slave replication. 
 
     `kubectl create -R -f datastores/`
```
Comment (mbreitung) [inline README.md]: we cannot have this run like this. the directories have changed and deployments and statefulsets are in the same dir. Both cannot be run on the same cluster

File: README.md [hunk 3]
```diff
@@ -51,49 +51,56 @@ Here are the most recent updates:
...
-* elasticsearch: stores event and metadata
-* cassandra: stores sysdig metrics
+* **mysql**: Stores user data and environmental data
+* **elasticsearch**: Stores events and metadata
+* **cassandra**: Stores Sysdig metrics
+
+**Notes**:
+
+- Backend components (worker, api and collector) are stateless deployed in deploymentsets
+- Datastores (redis, mysql, elasticsearch and cassandra) are stateful. They are configured in statefulsets that use
```
Comment (mbreitung) [inline README.md]: this is not true yet

File: README.md (hunk 2 above)
Comment (markyjackson-taulia) [reply README.md]: We should take out this entire section. Thoughts?

File: README.md (hunk 3 above)
Comment (markyjackson-taulia) [reply README.md]: @davideschiera can you remove this section too

**Comment by markyjackson-taulia** [comment]
We made some changes in our pr and incorporated your changes from your pr. Going to close this

**Pull request closed without merging**